.ruby-version

```diff
@@ -1 +1 @@
-2.3.6
+2.5.1
```

Comment:
@kgf , why do you say Travis is struggling to run 2.5.1?  **_Ostensibly_**, they support it on Ubuntu and Mac builds.

Reply:
@kgf so we're all on the same page: it looks like we're talking about https://travis-ci.org/ReadyResponder/ReadyResponder/builds/371111894 

Some googling turns this up: 
https://github.com/travis-ci/travis-ci/issues/8978

But im confused since it calls out rubygems 2.7.3, and the error is occuring on 2.7.6. The build is currently passing because the travis config of ruby 2.3x is overriding this `.ruby-version` file which declares 2.5.1. 

I can check the latest versions of bundler + rubygems @ mri 2.5.1 on travis if we want more information before continuing. But if things work fine in development and production on 2.5.1 its probably OK if travis runs using 2.3x, right?

Reply:
on second thought, it seems better to not change ruby and rails version simultaneously unless we need to. after discussing with @kgf it seems best to have us on mri2.3.6 across all environments until this lands and stabilizes.